src/nunjucks/moduk/components/tag/__tests__/macro.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, expect, it } from 'vitest'
```

Comment:
By default, `vitest` doesn't use globals. I think that's good (more explicit).

Reply:
I agree